src/components/address-book/AddressBookHeader/index.tsx

```diff
@@ -0,0 +1,67 @@
+import { Box, Button, SvgIcon } from '@mui/material'
+import type { ReactElement } from 'react'
+
+import Track from '@/components/common/Track'
+import { ADDRESS_BOOK_EVENTS } from '@/services/analytics/events/addressBook'
+import PageHeader from '@/components/common/PageHeader'
+import { ModalType } from '../AddressBookTable'
+import { useAppSelector } from '@/store'
+import { selectAllAddressBooks } from '@/store/addressBookSlice'
+import ImportIcon from '@/public/images/import.svg'
+import ExportIcon from '@/public/images/export.svg'
+import AddCircleOutlineIcon from '@mui/icons-material/AddCircleOutline'
+
+type Props = {
+  handleOpenModal: (type: ModalType) => () => void
+}
+
+const AddressBookHeader = ({ handleOpenModal }: Props): ReactElement => {
+  const allAddressBooks = useAppSelector(selectAllAddressBooks)
+  const canExport = Object.values(allAddressBooks).length > 0
+
+  return (
+    <PageHeader
+      title="Address book"
+      subtitle="Save frequent contacts for easy transacting"
+      action={
+        <Box display="flex" justifyContent="flex-end" pb={2}>
+          <Track {...ADDRESS_BOOK_EVENTS.IMPORT_BUTTON}>
+            <Button
+              onClick={handleOpenModal(ModalType.IMPORT)}
+              variant="text"
+              sx={{ color: 'text.primary' }}
+              startIcon={<SvgIcon component={ImportIcon} inheritViewBox />}
+            >
+              Import
+            </Button>
+          </Track>
+
+          <Track {...ADDRESS_BOOK_EVENTS.DOWNLOAD_BUTTON}>
+            <Button
+              onClick={handleOpenModal(ModalType.EXPORT)}
+              disabled={!canExport}
+              variant="text"
+              sx={{ color: 'text.primary' }}
+              startIcon={<SvgIcon component={ExportIcon} inheritViewBox />}
+            >
+              Export
+            </Button>
+          </Track>
+
+          <Track {...ADDRESS_BOOK_EVENTS.CREATE_ENTRY}>
+            <Button
+              onClick={handleOpenModal(ModalType.ENTRY)}
+              variant="text"
+              sx={{ color: 'text.primary' }}
```

Comment:
```suggestion
              color="primary"
```
It also makes the button color green in dark-mode which is in line with [Figma](https://www.figma.com/file/ptTs6lDBeUuLNySroJ5PiF/Web-Master-File?node-id=876%3A38473)